### PR TITLE
chore(update): update extension to look for rc branches

### PR DESCRIPTION
**Pull request by carmilx**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,50 +1,52 @@
...
+    document.querySelector('.merge-message details button[value=merge]').style.visibility = "hidden";
+    document.querySelector('.merge-message details button[value=rebase]').style.visibility = "hidden";
+
+  } else if (branch.startsWith('rc/')) {
+    // For PRs targeting main from rc/* branches, use Merge Commit
+    document.querySelector('.merge-message details button[value=merge]').click();
+    // Hide the other buttons
     document.querySelector('.merge-message details button[value=squash]').style.visibility = "hidden";
     document.querySelector('.merge-message details button[value=rebase]').style.visibility = "hidden";
-  } else {
```
Comment (denver) [inline script.js:12]: Do we want to keep the else block for branches that don't match the pattern, leaving options open?

**Review by brittgard — DISMISSED** [review]

File: script.js (hunk 1 above)
Comment (gnhieu) [reply script.js:12]: We should leave options open imo, I would rather not block people

File: script.js [hunk 2]
```diff
@@ -1,50 +1,52 @@
 /* global MutationObserver */
 
-function updateStrategy () {
+function updateStrategy() {
   const branch = document.querySelector('.head-ref').textContent;
```
Comment (mnrdrata) [inline script.js:4]: Do we want this to base .base-ref? .head-ref is the branch name being merged.

**Review by gnhieu — DISMISSED** [review]

File: script.js (hunk 2 above)
Comment (carmilx) [reply script.js:4]: let's keep it as is, we had this setting previously with no issues

**Pull request merged**